SECURITY.md

```diff
@@ -0,0 +1,22 @@
+# Security Policy
+
+To understand which version will receive security patches we recommend you to read the [Grails Support Schedule](https://grails.org/support-schedule.html).
+
+## Supported Versions
+
+Below you can see the versions of Grails currently being supported with security updates.
+
+| Version | Supported          |
+| ------- | ------------------ |
+| 5.2.x   | :white_check_mark: |
+| 5.1.x   | :x:                |
+| 5.0.x   | :x:                |
+| 4.1.x   | :white_check_mark: |
+| < 4.1   | :x:                |
+
+## Reporting a Vulnerability
+
+Please responsibly disclose (suspected) security vulnerabilities to
+**[The Grails Foundation](foundation@grails.org)**. You will receive a response from
+us within 48 hours. If the issue is confirmed, we will release a patch as soon
+as possible depending on complexity but historically within a few days.
```

Comment:
Not sure I would add the "but historically within a few days" part. I know we all know what that means, but I think some people would read that more stringently.

@JasonTypesCodes Your thoughts?